tutor/src/components/tabs.jsx

```diff
@@ -37,10 +38,12 @@ const Tabs = ({
 
   const selectTabIndex = (tab) => {
     tab = Number(tab);
+    if(pushToPath) {
     const query = extend(Router.currentQuery(windowImpl), { tab });
     history.push(
       windowImpl.location.pathname + '?' + qs.stringify(query)
-    );
+      );
+    }
     setActiveIndex(tab);
```

Comment:
@nathanstitt this is because we are using tabs in each of the offerings. so there is no need to have the `tab=` in the url. Each tab has its own logic.